frontend/src/components/preview/PreviewShortTexts.vue

```diff
@@ -55,15 +50,15 @@ const otherHeadnote = computed(() =>
 </script>
 
 <template>
-  <PreviewRow v-if="decisionName">
-    <PreviewCategory>{{ shortTextLabels.decisionName }}</PreviewCategory>
+  <PreviewRow v-if="decisionNames">
```

Comment:
```suggestion
  <PreviewRow v-if="decisionNames?.length">
```
Empty array ist truthy, deswegen brauchen wir hier vermutlich noch einen check gegen length